js/controllers/books.js

```diff
@@ -0,0 +1,112 @@
+/* global app */
+/* global angular */
+/*global $rootScope*/
+
+app.controller('booksCtrl', ['$scope', '$state', '$stateParams', 'bookService', 'authorService', 'enableEditService', 'uiGridConstants', function($scope, $state, $stateParams, bookService, authorService, enableEditService, uiGridConstants, $index) {
+  
+  $scope.include = function(obj) {
+     bookService.updateBooks(obj);
+   };
+   
+  $scope.authorNameDropdown = authorService.authorsList; 
+  $scope.bookTemp = bookService.booksList;
```

Comment:
this is not a meaningful name and should be refactored

Reply:
changed to bookData